docs/providers/google.mdx

```diff
@@ -9,8 +9,9 @@ import Map from '../components/Map';
 
 # Google Provider
 
-**note**: Google services require an API key. [Obtain here][1].
-For more options and configurations, see the [Google Maps developer docs][2].
+**<u>note</u>**: Google services require an API key enabled for both the Geocoding API and
```

Comment:
Would you believe that I was just working on adding this info, too? 😆 

If you think it's alright, I'd also like to add a recommendation about adding HTTP referrer restrictions as well.

Reply:
Sure! You are more than welcome to do that!

Reply:
Just added a few notes in 6c8d2d960ca028efe9100ef00d2d9a7dd63bacfd - let me know if those make sense, or please feel free to make any adjustments you feel are appropriate.

Reply:
No that's perfect! thanks!